cl/disclosures/tests.py

```diff
@@ -39,6 +43,18 @@ class DisclosureIngestionTest(TestCase):
         "JEF_format.pdf",
     )
 
+    @classmethod
+    def setUpTestData(cls) -> None:
+        judge = PersonWithChildrenFactory.create(id=2)
+        FinancialDisclosureFactory.create(
+            person=judge,
+            id=1,
+        )
+        FinancialDisclosureFactory.create(
+            person=judge,
+            id=2,
+        )
```

Comment:
I would refactor to remove IDs here. I actually don't think they're getting set by your factories anyway, since the factories themselves don't have ID fields. But like I was saying before, you don't want to set IDs in general. The better way to do so is to remove the IDs here, and to set the objects on the `cls` field so you can find them that way or look them up by other fields.

Reply:
√